README.md

```diff
@@ -8,12 +8,12 @@ This repo is designed to be used as a template for developing new data sets for
 It is currently set up for using `obs_test`.
 
 Relevant Files and Directories
------
+------------------------------
 path                  | description
 :---------------------|:-----------------------------
-`raw`                 | To be populated with raw data. Currently contains a single small fits file (taken from `obs_test`) to test `git-lfs` functionality.
-`calib`               | To be populated with master calibs. Currently empty.
-`templates`           | To be populated with calibrated images intended to be used as templates. Currently empty.
+`raw`                 | To be populated with raw data. May be organized however is convenient. Currently contains a single small fits file (taken from `obs_test`) to test `git-lfs` functionality.
```

Comment:
Out of context, it's not clear what "May be organized however is convenient" for `raw` means. I suppose it's in contrast to the specification below that `templates` must be organized as a filesystem-based Butler repo? It might be clearer to say that subdirectories are OK and that any images dumped in here will be ingested (assuming that's what you mean!). Same goes for `calib` description below.

Reply:
I want to keep it brief; how about "Subdirectories are allowed but not required."?